jenkins-agents/jenkins-agent-python/Dockerfile

```diff
@@ -1,8 +1,8 @@
-FROM quay.io/openshift/origin-jenkins-agent-base:4.7
+FROM quay.io/openshift/origin-jenkins-agent-base:4.9
```

Comment:
4.9 isn't released, so I wouldn't suggest using that tag.

Reply:
Thanks. Where can I go to find the latest release, I've been using https://quay.io/repository/openshift/origin-jenkins-agent-base?tag=latest&tab=tags

Reply:
I'd typically go to:
- https://docs.openshift.com/container-platform

currently, 4.7 is the latest (_4.8 will be out soon_)

Reply:
thanks. I will close this PR.